Give `expectMsg` some more time in CoordinatedShutdownShardingSpec

raboof opened the pull request:
The 1-second deadline was missed on jenkins, https://jenkins.akka.io:8498/blue/rest/organizations/jenkins/pipelines/akka-nightly/runs/6161/log/?start=0

Refs #26214

patriknw commented on akka-cluster-sharding/src/test/scala/akka/cluster/sharding/CoordinatedShutdownShardingSpec.scala:
```diff
@@ -76,13 +76,13 @@ class CoordinatedShutdownShardingSpec extends AkkaSpec(CoordinatedShutdownShardi
       region2.tell(3, p3.ref)
-      p3.expectMsg(1.seconds, 3)
+      p3.expectMsg(3)
     }, 10.seconds)
```
I could imagine that the reason for the shorter timeouts are because they are inside `awaitAssert` and if failing they will be retried more quickly. Perhaps increase the 10.seconds instead?

raboof commented:
Hmm the same error, even with 20 seconds. Perhaps there's really something going on I suppose :)

patriknw commented:
Why did it suddenly start failing now? 3 failures in akka-nightly: https://jenkins.akka.io:8498/blue/organizations/jenkins/akka-nightly/activity/

raboof committed:
Add more logging to explain test failure

raboof committed:
Widen timeout of first ping

raboof committed:
Give each message 3 seconds to roundtrip

Otherwise the chance that all 3 messages arrive within 1 second is low on
current jenkins

raboof committed:
Add debug logging

raboof commented:
I'd like to see it fail :D

raboof commented:
.PLS BUILD

raboof commented:
PLS BUILD

raboof commented:
PLS BUILD

raboof commented:
PLS BUILD

raboof commented:
PLS BUILD

raboof commented:
Weird that it seem to work reliably again in this PR. Perhaps we should consider merging but leaving the logging in, so we can diagnose further when it fails again even with 3-seconds timeouts?

patriknw commented:
I found the real cause.
Superseded by https://github.com/akka/akka/pull/26242
